### ISSUE TITLE
simulation

### ISSUE THREAD

**Issue by daohu527** [daohu527/carla_bridge#1]

**Comment by marcusvinicius178** [comment]
Hi, Mr @daohu527 is this Carla bridge already working with Apollo? If yes which Apollo version? Would you have a tutorial? 

Greetings,

Marcus

**Comment by daohu527** [comment]
No, **I'll finish the project in December**, and in general should support the latest version, other version will also be ok, I think. 

This project will directly refer to ros_carla_bridge, because we already have pycyber.

**Comment by YuqiHuai** [comment]
Hi @daohu527, hope you are doing well! How are things going with this project?

**Comment by daohu527** [comment]
I'm a bit busy recently, so the project will be delayed for about 2 months, I need to ensure that the test is completed

**Comment by marcusvinicius178** [comment]
Hi @daohu527  I have read something about Carla- Apollo bridge: https://carla.org/2022/11/28/carla-apollo-bridge/
Their GitHub (I have still did not tried) :https://github.com/guardstrikelab/carla_apollo_bridge

Do you know if this integration is already done, stable, and complete?
Because as we discussed earlier, The **AuroAI** was not "complete" in terms of Perception and Localization:

**localization**:  [AuroAI/carla_apollo_bridge repository](https://github.com/AuroAI/carla_apollo_bridge/issues/19)
**Perception** [ApolloAuto/repository](https://github.com/ApolloAuto/apollo/issues/14338)

Do you know if this **Guardstrike**'s bridge is complete and working?
In addition, is your bridge already released, did it work (just curious)?

**Comment by daohu527** [comment]
I'm not sure, I browsed the codes, and their ideas are the same as mine, that is, to transplant ros bridge to cyber. I think you can ask questions in their repo.

> In addition, is your bridge already released, did it work (just curious)?

I am currently refactoring the perception module. In my spare time, I am mainly reading track papers and reading some books by Kazuo Inamori.

But in general, I will finish it this year. You know that open source projects are not just started, but maintained for a long time

**Comment by marcusvinicius178** [comment]
Hi @daohu527 ,

I hope you're doing well. I'm reaching out to inquire about the current status of the CARLA bridge project. Myself and a master's student are keen to use it for our academic research.

Could you kindly let us know if the project is operational and if there are plans for ongoing maintenance or development? Any brief info on this would greatly assist in our planning. If it's convenient, could you share if there's a basic demo or tutorial available that could help me get started? 

I mean I prefer using Apollo than Autoware, that's why taking a look here. 

Thanks for your time and effort on this project.

Best regards,

Marcus

**Comment by daohu527** [comment]
Sorry, I have been doing some other work recently, and I don’t have much time to take care of so many projects at the moment. Of course, there is already a mature project here. You can try

https://github.com/guardstrikelab/carla_apollo_bridge

Also I will continue this project, my goal is to make good wheels
